test/regressions/fixtures/CssVarsProvider/CssVarsProviderJoy.js

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { CssVarsProvider } from '@mui/joy/styles';
+import Box from '@mui/joy/Box';
+import Button from '@mui/joy/Button';
+
+export default function VariantColorJoy() {
```

Comment:
```suggestion
// All buttons should look the same, otherwise it means that multiple instances with different prefix does not work.
export default function VariantColorJoy() {
```